tailwind.config.js

```diff
@@ -90,11 +90,16 @@ module.exports = {
         }
       }
     },
+    maxHeight: {
+      '32': '8rem'
+    }
+  },
```

Comment:
turns out max-height doesn't have values you'd want, so I added my own!

Reply:
🚩 Worth checking out these docs, as this implementation overrides all the `max-height` values, rather than adding your new value to them. You can extend it though 👉 https://tailwindcss.com/docs/theme/#extending-the-default-theme

Reply:
It'd be nice just to have all our rem values available here too. Bit sad that tailwind doesn't do that for us!

Reply:
Yeah I think I'll add them all onto our theme